app/experimenter/experiments/api/v5/query.py

```diff
@@ -1,31 +1,39 @@
 import graphene
 
-from experimenter.experiments.api.v5.types import NimbusExperimentType
+from experimenter.experiments.api.v5.types import (
+    NimbusExperimentIdOrder,
+    NimbusExperimentOptionalStatus,
+    NimbusExperimentType,
+)
 from experimenter.experiments.models.nimbus import NimbusExperiment
 
 
 class Query(graphene.ObjectType):
-    experiments_by_status = graphene.Field(
-        graphene.List(NimbusExperimentType),
-        status=graphene.Argument(
-            NimbusExperimentType._meta.fields["status"]._type, required=False
-        ),
-    )
-    all_experiments = graphene.List(
+    experiments = graphene.List(
         NimbusExperimentType,
+        description="List Nimbus Experiments.",
+        offset=graphene.Int(),
+        limit=graphene.Int(),
+        order=NimbusExperimentIdOrder(),
+        status=NimbusExperimentOptionalStatus(),
```

Comment:
So to filter on a field on the model you have to explicitly declare it here?

Reply:
Alternatively would be more effective for GQL just send all experiments to the frontend and the filtering/sorting be done on the client side?

Reply:
It's possible to reach into the `NimbusExperimentType` object, but because those types are generated off the model, the status becomes a required field. In this case we want the status to be an optional argument so using the generated required graphene type doesn't work. I spent quite a bit of time trying to grab the object and manipulate it back to optional for use here, but that didn't work out.

Reply:
Ah ya I see 👍